src/java/us/kbase/templates/module_dockerfile.vm.properties

```diff
@@ -2,13 +2,24 @@ FROM kbase/kbase:sdkbase.latest
 MAINTAINER #if($username)${username}#{else}KBase Developer#{end}
 
 # -----------------------------------------
-
-# Insert apt-get instructions here to install
-# any required dependencies for your module.
+# In this section, you can install any system dependencies required
+# to run your App.  For instance, you could place an apt-get update or
+# install line here, a git checkout to download code, or run any other
+# installation scripts.  Here we install a python coverage tool and an
+# https library that is out of date in the base image.
 
 # RUN apt-get update
 #if( $language == "python")
+
 RUN pip install coverage
+
+# update security libraries in the base image
+RUN pip install cffi --upgrade \
+    && pip install pyopenssl --upgrade \
+    && pip install ndg-httpsclient --upgrade \
```

Comment:
They're up to date for now. They won't be in the future, and my feeling was we should keep the security stuff as new as possible.